nexus/db-model/src/physical_disk.rs

```diff
@@ -85,3 +85,75 @@ impl DatastoreCollectionConfig<super::Zpool> for PhysicalDisk {
     type CollectionTimeDeletedColumn = physical_disk::dsl::time_deleted;
     type CollectionIdColumn = zpool::dsl::sled_id;
 }
+
+mod diesel_util {
```

Comment:
This mimics the `ApplySledFilterExt` trait pretty explicitly:

https://github.com/oxidecomputer/omicron/blob/30b6713305faa57e80b25089a66c0b99f0553ca7/nexus/db-model/src/sled.rs#L371